.github/workflows/test.yml

```
@@ -33,21 +33,13 @@ jobs:
   pwa-kit:
     strategy:
       matrix:
-        node: [14, 16, 18]
-        npm: [6, 7, 8, 9]
-        # We exclude the less likely Node and NPM combinations to reduce the number of GA builds.
-        exclude:
-          - node: 16
-            npm: 6
-          - node: 18
-            npm: 6
-          - node: 14
-            npm: 9
+        node: [16, 18]
+        npm: [7, 8, 9]
     runs-on: ubuntu-latest
     env:
       # The "default" npm is the one that ships with a given version of node.
       # For more: https://nodejs.org/en/download/releases/
-      IS_DEFAULT_NPM: ${{ matrix.node == 14 && matrix.npm == 6 || matrix.node == 16 && matrix.npm == 8 }}
+      IS_DEFAULT_NPM: ${{ matrix.node == 16 && matrix.npm == 8 }}
```

```suggestion
      IS_DEFAULT_NPM: ${{ matrix.node == 16 && matrix.npm == 8 || matrix.node == 18 && matrix.npm == 9 }}
```
We use `IS_DEFAULT_NPM` to decide if we run the _Update NPM version_ step
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/1f444a24db0f81de7b3038c37ef67ff55b13b84e/.github/workflows/test.yml#L54-L57